src/components/Atoms/Logo/Logo.js

```diff
@@ -37,7 +35,9 @@ const Logo = ({ rotate, sizeSm, sizeMd, campaign }) => {
       <Image
         src={campaign === 'comicrelief' ? crLogo : srLogo}
         alt={
-          campaign === 'comicrelief' ? 'Comic Relief logo' : 'Sport Relief logo'
+          campaign === 'comicrelief'
+            ? 'Comic Relief logo - home page'
```

Comment:
@LouiaR sorry, this was a recommendation from DAC report. But thinking better I don't think it make sense to use home page here.. this logo is also used in the footer. Let's just use `Comic Relief logo` and on CR we add `title="homepage"` to the link and a visually hidden span.

Reply:
@gusliedke what about add `alt='comicrelief.com'`